nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/hosted/RHostedFacetImpl.java

```diff
@@ -125,6 +125,6 @@ protected void doPutPackagesGz(final StorageTx tx, final String basePath, final
   {
     RFacet rFacet = facet(RFacet.class);
     Asset asset = rFacet.findOrCreateAsset(tx, buildPath(basePath, PACKAGES_GZ_FILENAME));
-    saveAsset(tx, asset, tempPackagesGz, "", null);
+    RFacetUtils.saveAsset(tx, asset, tempPackagesGz, "", null);
```

Comment:
use static import

Reply:
Done in dc77f851ee8368df968ded268279ea0320e9d2bc